witness/src/Dscp/Snowdrop/Configuration.hs

```diff
@@ -91,13 +97,15 @@ data Values
     = TipValueVal       (TipValue HeaderHash)
     | BlundVal           SBlund
     | AccountOutVal      Account
+    | TxVal              T.GTxWitnessed
```

Comment:
This way we store all transactions twice. Maybe it's better to define mapping `tx -> headerHash`?

Reply:
Yeah, right, we also want to return block reference for `/transactions` endpoint anyway.